source/css/scss-common/_bootstrap.scss

```diff
@@ -5,12 +5,12 @@
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
  */
 
-$cursor-disabled: disabled;
+$cursor-disabled: not-allowed;
 
 @import './../../node_modules/bootstrap/scss/functions';
 @import './../../node_modules/bootstrap/scss/variables';
 @import './../../node_modules/bootstrap/scss/mixins';
-@import './../../node_modules/bootstrap/scss/root';
+// @import './../../node_modules/bootstrap/scss/root';
```

Comment:
Commented line, probably something to remove?

Reply:
No, it is from the bootstrap master scss file, so I want to keep the reference.